english/explore-outside/talk-child/talk-more/nice-talk/help-child/abandon-hope/abandon-hope.md

```diff
@@ -0,0 +1,7 @@
+You swim ashore and look back. The yellow lake is quiet. Even the marshmallow boat has melted away and the only proof of there ever having existed a green sad child with a thick Pittsburgh accent is now the image in the back of your mind. Wondering about the child, you look at the car keys in your hand.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.